miio/viomivacuum.py

```diff
@@ -167,17 +224,27 @@ class ViomiWaterGrade(Enum):
     High = 13
 
 
-class ViomiMopMode(Enum):
+class ViomiRoutePattern(Enum):
     """Mopping pattern."""
 
     S = 0
     Y = 1
 
 
+class ViomiVoiceState(Enum):
+    Off = 1
+    On = 5
```

Comment:
If this has just two states, it'd be better to use a simple boolean.

Reply:
Like
```
    @property
    def voice_state(self) -> bool:
        """Voice state."""
        return self.data["v_state"] == 5
```
?

Reply:
Fixed